src/datasets/builder.py

```diff
@@ -493,7 +493,7 @@ def _create_builder_config(
         )
         is_custom = (config_id not in self.builder_configs) and config_id != "default"
         if is_custom:
-            logger.warning(f"Using custom data configuration {config_id}")
+            logger.info(f"Using custom data configuration {config_id}")
```

Comment:
I did this because I think it's not relevant anymore, and because I find it confusing to show this when calling `IterableDataset.from_generator`